src/clusterfuzz/_internal/cron/cleanup.py

```diff
@@ -1056,6 +1059,19 @@ def update_component_labels_and_id(policy, testcase, issue):
   component_id = _get_predator_result_item(testcase,
                                            'suspected_buganizer_component_id')
 
+  # These bugs were filed via the direct upload and hence specific to the
+  # google issue tracker.
+  if hasattr(issue,
```

Comment:
when would it not have component_id?
You can replace this with getattr(issue, 'component_id', None) == VRP_UPLOAD_COMPONENT_ID.

Reply:
i believe an issue wouldnt have component_id for non google issue tracker (monorail, jira) 

but ooh thank you done!